app/models/user.rb

```diff
@@ -41,4 +50,13 @@ def card_for_review
   def has_linked_github?
     authentications.where(provider: 'github').present?
   end
+
+  def self.notify_users_with_reviews_available
+    for_review_counters_at(Time.now).each do |user|
```

Comment:
Do not use `Time.now` without zone. Use one of `Time.zone.now`, `Time.now.current`, `Time.now.in_time_zone`, `Time.now.utc`, `Time.now.getlocal`, `Time.now.iso8601`, `Time.now.jisx0301`, `Time.now.rfc3339`, `Time.now.to_i`, `Time.now.to_f` instead.